FORMATTING_GUIDE.md

```diff
@@ -0,0 +1,314 @@
+
+To take high-resolution screenshots, in Firefox, right-click on the page and choose “Take Screenshot”.
+
+Image borders are automatic; do not manually add a border to an image.
+
+Always **place an image on a new line**—not next to text. Otherwise, the text might render as aligned to the bottom of the image, with the image on the right. If the image is under a list item, place it on a new line with a tab. Don’t insert artificial line breaks using `<br>`.
+
+## Labels
+
+You can use the following labels:
+
+* label-blue 
+* label-green
+* label-purple 
+* label-red 
+* label-yellow 
+
+Use a purple label to specify in which version an API was introduced.
+
+**Example**:
+
+```
+# Alias
+Introduced 1.0
+{: .label .label-purple }
+```
+
+## Callouts
+
+You can use four levels of callouts:
+
+*  `{: .note}` blue
+*  `{: .tip }` green
+*  `{: .important}` yellow
+*  `{: .warning}` red
+
+Place a callout directly under the paragraph to which you want to apply the callout style.
+
+**Example**: 
+
+```
+In case of a cluster or node failure, all PIT data is lost.
+{: .note}
+```
+
+For a callout with multiple paragraphs or lists, use `>`:
+
+```
+>   ****PREREQUISITE****
+>
+>   To use a custom vector map with GeoJSON, install these two required plugins:
+>   * OpenSearch Dashboards Maps [`dashboards-maps`](https://github.com/opensearch-project/dashboards-maps_) front-end plugin
+>   * OpenSearch [`geospatial`](https://github.com/opensearch-project/geospatial_) backend plugin
+{: .note}
+
+```
+
+## Buttons
+
+You can use either `copy` or `copy-curl` includes for code snippets. The `copy` include places a **Copy** button on the code snippet, while the `copy-curl` include places both **Copy** and **Copy as cURL** buttons. Use the `copy-curl` include for API requests. If an API request is already in the cURL format, use the `copy` include.
+
+**Example of a `copy` include:**
+
+````
+```bash
+curl -XGET "localhost:9200/_tasks?actions=*search&detailed
+```
+{% include copy.html %}
+````
+
+
+**Example of a `copy-curl` include:**
+
+````
+```json
+PUT /sample-index1/_clone/cloned-index1
+{
+  "aliases": {
+    "sample-alias1": {}
+  }
+}
+```
+{% include copy-curl.html %}
+````
+
+## Adding pages or sections
+
+When adding a page or a section, make the `nav_order` of the child pages multiples of 10. For example, if you have a parent page `Clients`, make child pages `Java`, `Python`, and `JavaScript` have a `nav_order` of 10, 20, and 30, respectively. Doing so makes inserting child pages easier.
+
+## Videos
+
+To insert a video, add a youtube player include similar to the following:
+
+```
+{% include youtube-player.html id='_g46WiGPhFs' %}
+```
+
+Note that the `id` variable refers to the youtube video ID that is the last part of the URL. For example, the youtube video at the URL `https://youtu.be/_g46WiGPhFs` has the id `_g46WiGPhFs`. The ID must be surrounded with single quotation marks.
```

Comment:
Sounds a little off to me at least. Maybe like "the `id` variable refers to the youtube video ID at the end of the URL" ?